tests/test_main.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from unittest.mock import Mock
+from omegaconf import OmegaConf, DictConfig
+from stac_mjx import utils
+from stac_mjx import main
+
+
+@pytest.fixture
+def mock_omegaconf_load(monkeypatch):
+    mock = Mock(
+        side_effect=[
+            OmegaConf.create({"model": "config"}),
+            OmegaConf.create({"stac": "config"}),
+        ]
+    )
+    monkeypatch.setattr(OmegaConf, "load", mock)
+    return mock
+
+
+@pytest.fixture
+def mock_init_params(monkeypatch):
+    mock = Mock()
+    monkeypatch.setattr(utils, "init_params", mock)
+    return mock
```

Comment:
Do you think these will be useful for other tests, can you please put these in a file a `fixtures` dir. in tests.

Reply:
I'm not sure if they'll be useful for other tests because testing things deeper in the logic will require actual param values, but will move to a fixtures dir.